index.html

```diff
@@ -29,8 +28,8 @@
       
       <section id="port-without-header">
          <div class="port-contenido">
-            <h2 id="tit-port" class="title">I’m Ivan Glad to see you!</h2>   
-            <p id="intro-text-1" class="parrafo">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            <h2 id="tit-port" class="title">I’m Ivan <br> Glad to see you!</h2>   
+            <p id="intro-text-1" class="parrafo">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!  If you like what you see and have a project you need coded, don’t hestiate to contact me.
             </p>   
             <h3><a href="https://www.linkedin.com/in/ivan-martinez-von-halle/" class="connect-text">LET'S CONNECT</a></h3>
             <ul class="social-media-icons">
```

Comment:
- [ ] Great layout you got @ivanmvh, The thing to add here will be the background image as shown in the Figma design. 

### Your design
![your](https://user-images.githubusercontent.com/69077061/193392723-416fa187-08ba-4413-aa69-16d3f9f4513f.PNG)

### [Figma design  ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
![figma](https://user-images.githubusercontent.com/69077061/193392780-06f36871-c53a-4b34-81bd-45bf6e6eb84b.PNG)

Reply:
I define the image  here:
![image](https://user-images.githubusercontent.com/817718/193398280-e670812c-78bd-455a-9ef4-c73d8724e347.png)